### ISSUE TITLE
Facebook Adapter not found 

### ISSUE THREAD

**Issue by DavidAPears** [ammarahm-ed/react-native-admob-native-ads#334]
I know this has been reported before and marked as closed, but there still seems to be an issue and I can't work out how to resolve it. This seems to be THE library for using Native AdMob with react native, so any help would be hugely appreciated.

XCode (latest version): 14.3
react-native: "0.71.6"
react-native-admob-native-ads: ^0.6.0
react-native-vector-icons: ^9.2.0

When I add the following to my podfile (as per docs)

```
pod 'Google-Mobile-Ads-SDK'
pod 'GoogleMobileAdsMediationFacebook'
```

And then `pod install --repo-update`

Pod installation works, however, running the project in XCode gives the error:

`Framework not found FacebookAdapter`

Sadly, previous fixes and suggestions do not resolve this issue. 

<img width="343" alt="Screenshot 2023-04-14 at 10 21 40" src="https://user-images.githubusercontent.com/36195948/231987339-635c366e-ade0-4d15-811b-5c98b0afa34e.png">

**Comment by fedeinfe** [comment]
same here... do you have some news?

**Comment by cddceo** [comment]
downgrade "react-native-google-mobile-ads": "9.1.2",

podfile
pod 'GoogleMobileAdsMediationFacebook', '6.11.0.0'

**Comment by ihdev524** [comment]
@cddceo doesn't work for me

**Comment by fedeinfe** [comment]
@ihdev524 @DavidAPears 
check this for the solution https://github.com/ammarahm-ed/react-native-admob-native-ads/issues/324

**Comment by ammarahm-ed** [comment]
This has been fixed in v0.6.3. Please update.
